src/jdk.javadoc/share/classes/jdk/javadoc/internal/doclets/toolkit/util/VisibleMemberTable.java

```diff
@@ -598,6 +602,87 @@ boolean allowInheritedMethods(ExecutableElement inheritedMethod,
         return true;
     }
 
+    // Check whether the signature of an overriding method has any changes worth
+    // being documented compared to the overridden method.
+    private boolean overridingSignatureChanged(ExecutableElement method, ExecutableElement overriddenMethod) {
```

Comment:
In the body of this method, can you add in:

change in abstract/non-abstract (both ways)
change in final/non-final